ecosystem_tests/ecosystem_tests_cli/commands/prepare_test_manager.py

```diff
@@ -0,0 +1,55 @@
+########
+# Copyright (c) 2014-2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ..utilities import prepare_test_env
+from ...dorkl.runners import prepare_test_dev
+from ...ecosystem_tests_cli import ecosystem_tests
+from ..secrets import prepare_secrets_dict_for_prepare_test
+
+
+@ecosystem_tests.command(name='prepare-test-manager',
+                         short_help='Prepare test manager(licence, '
+                                    'secrets, etc.)')
+@ecosystem_tests.options.license
+@ecosystem_tests.options.secret
+@ecosystem_tests.options.file_secret
+@ecosystem_tests.options.encoded_secrets
+@ecosystem_tests.options.plugin
+@ecosystem_tests.options.plugins_bundle
+@ecosystem_tests.options.no_bundle
+@ecosystem_tests.options.container_name
+def prepare_test_manager(license,
+                         secret,
+                         file_secret,
+                         encoded_secret,
+                         plugin,
+                         bundle_path,
+                         no_bundle_upload,
+                         container_name):
+    """
+    This command responsible for prepare test manager.
+    """
+    with prepare_test_env(license,
+                          secret,
+                          file_secret,
+                          encoded_secret,
+                          container_name):
```

Comment:
Why should `prepare_test_env` be a context manager and not a wrapper?

Reply:
I made it context manager because I think its a classic use of context manager to acquire resource(in this case the the test env vars) then free the resource after executing all I need.
https://github.com/cloudify-incubator/cloudify-ecosystem-test/pull/80/files#diff-0e863ba1f417aa2aebbd49dd722e16363597e9fcf6a15f49f8307d2049dd9b16R39